### DISCUSSION TITLE
유니티로 월드 제작시 범위는 얼마로 해야하나요?

### DISCUSSION THREAD

**Discussion by hansolys** [naverz/zepeto-studio-kor#498] [5. Q&A]
유니티로 제페토 월드만드실때 유니티로 범위는 어느정도 잡고작업하시는지 궁금합니다.

정해진 크기나 범위가 있는지 궁금합니다!

완전초보라서 잘모르겠습니다.ㅠㅅㅠ

**Comment by hipsc159** [comment]
범위는 상관없고 제페토 최대 용량이 500mb 인것만 조심하시면 됩니다.

**Reply by hansolys** [reply]
감사합니다!

**Reply by hansolys** [reply]
혹시 용량은 어느정도나오는지 어떻게 확인할수있을까요?

**Reply by Zepeto-TechEvangelist** [reply]
빌드를 수행한 직후에 생성되는 파일 정보에서 전체적인 용량을 확인할 수 있습니다.
각 부분별로 얼마나 용량을 차지하는지 알고 싶다면
Console 창으로 이동(메뉴: Window > General > Console)하여 오른쪽 상단의 작은 드롭다운 패널을 클릭하고 Open Editor Log 를 선택해 주세요.
